### PR TITLE
updating existing SBOMs and Attestations doc due to product changes

### LINKED ISSUE #842: Recycle existing SBOMs and Attestations doc
[SBOMs and Attestations]() doc will be going stale with the new features coming to Enforce. This issue is to track the work to recycle the content elsewhere.

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

I realized with the new SBOM features that the existing SBOMs and Attestations doc will be stale. Ville suggested recycling this content to limit the mentions of Enforce and focus on the differences between SBOMs and ATtestations.

To that end, I cut any discussion of how Enforce ingests SBOMs (now covered [here](https://edu.chainguard.dev/chainguard/chainguard-enforce/sboms/sboms-in-enforce/)) and made the article more generic overall.

### What should this PR do?

resolves #842 

### How should this PR be tested?

I'm not 100% certain the Open Source section is the best spot for this doc, but it made sense since there's already an SBOM bucket there. I'm open to moving it. Also, please give the shortened article a read to make sure it still makes sense and is valuable.

### CODE REVIEW

**Review by ltagliaferri — COMMENTED** [review]
NGINX config should update the links so people don't get lost:

https://edu.chainguard.dev/chainguard/chainguard-enforce/sboms/ should point to /chainguard/chainguard-enforce/sboms-in-enforce/

and 

https://edu.chainguard.dev/chainguard/chainguard-enforce/sboms/sboms-and-attestations/ should point to /open-source/sbom/sboms-and-attestations/

File: content/open-source/sbom/sboms-and-attestations.md [hunk 1]
```diff
@@ -1,44 +1,29 @@
...
+One of the first steps to improving your software supply chain security is to establish a process for creating quality *Software Bills of Materials* (SBOMs). An [SBOM](/open-source/sbom/) is a formal record that contains the details and supply chain relationships (such as dependencies) of the components used in building software.
 
-[Cosign](/open-source/sigstore/cosign/an-introduction-to-cosign/)  — a part of the Sigstore project — supports software artifact signing, verification, and storage in an [OCI (Open Container Initiative)](/open-source/oci/) registry. The `cosign` command line tool offers two subcommands that you can use to associate an SBOM with a container image and then upload them to a registry: `cosign attach` and `cosign attest`. 
+[Cosign](/open-source/sigstore/cosign/an-introduction-to-cosign/)  — a part of the Sigstore project — supports software artifact signing, verification, and storage in an [OCI (Open Container Initiative)](/open-source/oci/) registry. The `cosign` command line tool offers two subcommands that you can use to associate an SBOM with a container image and then upload them to a registry: `cosign attach` and `cosign attest`.
```
Comment (ltagliaferri) [inline content/open-source/sbom/sboms-and-attestations.md:24]:
```suggestion
[Cosign](/open-source/sigstore/cosign/an-introduction-to-cosign/) — a part of the Sigstore project — supports software artifact signing, verification, and storage in an [OCI (Open Container Initiative)](/open-source/oci/) registry. The `cosign` command line tool offers two subcommands that you can use to associate an SBOM with a container image and then upload them to a registry: `cosign attach` and `cosign attest`.
```

**Review by ltagliaferri — APPROVED** [review]
This is great, thanks for cleaning this up and moving it over. It is well suited for the SBOM section. I would say it could be cool to link to the Cosign SBOM tutorial: https://edu.chainguard.dev/open-source/sigstore/cosign/how-to-sign-an-sbom-with-cosign/

**Comment by SharpRake** [comment]
> This is great, thanks for cleaning this up and moving it over. It is well suited for the SBOM section. I would say it could be cool to link to the Cosign SBOM tutorial: https://edu.chainguard.dev/open-source/sigstore/cosign/how-to-sign-an-sbom-with-cosign/

Ah! forgot to add that link, added in new commit.

**Pull request merged**